### PR TITLE
add BatchNormTraining and BatchNormGrad support in hlo-legalize-to-lhlo

### PR DESCRIPTION
**Pull request by lchang20**
It basically resolved lowering of some op with a result as a tuple of tensors.
There seem only BatchNormTraining and BatchNormGrad requiring this.

### CODE REVIEW

**Comment by lchang20** [comment]
@googlebot I signed it!

**Comment by joker-eph** [comment]
Sorry, we don't accept pull-request in this repository: it is a read-only view of https://github.com/tensorflow/tensorflow/tree/master/tensorflow/compiler/mlir/hlo 
Do you mind send a pull-request there?
